### PR TITLE
Test UK localozation with transifex

### PR DESCRIPTION
**Pull request by MaxymVlasov**
100% reviewed for the source file 'content/en/releases/notes.md'
on the 'uk' language.

### CODE REVIEW

**Comment by utkarsh-singh1** [comment]
Hi @MaxymVlasov, please follow these guides to sign easycla,

You can find about EasyCLA here - https://docs.linuxfoundation.org/lfx/easycla

and the steps to sign EasyCLA are here - https://docs.linuxfoundation.org/v2/easycla/contributors/individual-contributor

You can start your signing EasyCLA from here - https://api.easycla.lfx.linuxfoundation.org/v2/repository-provider/github/sign/18706487/51478266/38056/#/?version=2

File: content/uk/releases/notes.md [hunk 1]
```diff
@@ -0,0 +1,13 @@
+---
+linktitle: Примітки до релізу
+title: примітки
+type: документація
```
Comment (sftim) [inline content/uk/releases/notes.md:4]: Did you mean:
```suggestion
type: documentation
```

File: content/uk/releases/notes.md [hunk 2]
```diff
@@ -0,0 +1,13 @@
...
+description: >
+Примітки до релізу Kubernetes
```
Comment (sftim) [inline content/uk/releases/notes.md:6]:
```suggestion
  Примітки до релізу Kubernetes
```

**Review by sftim — COMMENTED** [review]
Some early feedback

**Comment by MaxymVlasov** [comment]
@sftim That's a test PR made just for test Transifex integration ad didn't contain any valid translation (at least, it was not reviewed at all and was made to have an EasyCLA comment which I can refer to newcomers and in the Transifex support chat.)

Feel free to ignore it, it will be closed when we find the best, working solution for us.
And, as we are already here, be prepared that we will go to you in Slack in a week or so to discuss the localization process and integrations :)

**Pull request closed without merging**